### ISSUE TITLE
--disable-machine-id --disable-encryption-win flags doesn't work

### ISSUE THREAD

**Issue by graphixillusion** [Alex313031/Thorium-Win#26]
Hi there. 

As the title says i've tried to add this two flags to the batch but when i install some extension and then move everything on a different computer, the extension is gone. How to make this fully portable (with everything, extensions, settings, etc...)? Thank you!

**Comment by Alex313031** [comment]
@graphixillusion This is because Thorium doesnt yet have the patches from ungoogled chromium that allow this. The flags are there, but they are ignored in source code.

**Comment by graphixillusion** [comment]
@Alex313031 thank you for the answer. Is there any eta for this patches to become availablein thorium too? Without these two flags is not a real portable app sadly...

**Comment by Alex313031** [comment]
@graphixillusion That is strange though. When I use the portable linux or windows version, it will sign me out of sites, and remove all my passwords, but extensions and app settings are preserved.

**Comment by graphixillusion** [comment]
@Alex313031 i'll try again with the last version then and report it back

**Comment by graphixillusion** [comment]
@Alex313031 I've just tried the last version but is the same: it keeps the settings but extensions are reset when you move the folder on a different system

**Comment by Alex313031** [comment]
@graphixillusion I'm going to add these ungoogled patches to enable truly portable usage

**Comment by Alex313031** [comment]
@graphixillusion Added here > https://github.com/Alex313031/thorium/commit/002832e67537bddbe0cdceaafe7986c6ba2238a0

Ive also added these cmdline flags to the thorium.bat for the portable zips, so it should just work out of the box.

Also of note, the --disable-encryption-win was changed to just --disable-encryption in ungoogled chromium

**Comment by Alex313031** [comment]
@graphixillusion Is it working for you?

**Comment by graphixillusion** [comment]
> @graphixillusion Is it working for you?

Yep i just tried it and i confirm that it works as it should, thank you
